src/config.rs

```diff
@@ -5,6 +5,7 @@ use serde::{Deserialize, Serialize};
 use crate::{util::ReadableSize, Result};
 
 const MIN_RECOVERY_READ_BLOCK_SIZE: usize = 512;
+const MIN_RECOVERY_THREADS: usize = 2;
```

Comment:
Now `recovery_threads` is not the actual thread count. Speak accurately, it's how many segments we split the total log files. The min segment count is 2 because either append log or rewrite log can be split to at least one segments. 

I agree that we need to refine the config. I will make the config control the actual threads that rayon use and split log segments based on that.

Reply:
Okay, just want to mention that threads=1 should be supported, and as a side note, Append and Rewrite should have different concurrency based on their actual size.